files/views.py

```
@@ -496,7 +508,254 @@ def edit_media(request):
+                        messages.add_message(
+                            request,
+                            messages.ERROR,
+                            "Failed to prepare media for re-encoding",
+                        )
+                        return HttpResponseRedirect(media.get_absolute_url())
+                finally:
+                    # Always close the temporary file handle on all paths (including early returns)
+                    if temp_file_handle:
+                        try:
+                            temp_file_handle.close()
+                        except Exception as close_error:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)